ncp.sh

```diff
@@ -131,8 +132,27 @@ EOF
 
   cat > /home/www/ncp-launcher.sh <<'EOF'
 #!/bin/bash
+
+declare -a args
+for arg in $@
+do
+  if [[ $arg == "--auto-attach" ]]
```

Comment:
we are not using these options, are we?

Reply:
I _was_ using them previously from ncp-launcher.php. But not anymore no. I should probably remove those, but they work so neatly :see_no_evil:

Reply:
:laughing: yeah better to simplify if we are not using them

Reply:
Implemented in aaa4202d2bb8d9360a66c2b22703630ada44868c